src/components/JdsPagination/Pagination.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div
+    :class="{
+      'jds-pagination font-sans-1': true,
+      'jds-pagination--disabled': disabled,
+    }"
+  >
+    <div class="jds-pagination__wrapper">
+      <!-- left section -->
+      <div class="jds-pagination__page-control--left">
+        <div class="jds-pagination__page-control__select">
+          <span>
+            Tampilkan
+          </span>
+          <select :value="mItemsPerPage" @change="onItemsPerPageChange" :disabled="disabled">
+            <option :value="item" v-for="item in itemsPerPageOptions" :key="item">
+              {{ item }}
+            </option>
+          </select>
+          <span>
+            item
+          </span>
+          <i class="jds-pagination__divider" />
+          <span>dari total <strong>{{ totalRows }}</strong></span>
+          <i class="jds-pagination__divider" />
+        </div>
+      </div>
+
+      <!-- rigth section -->
+      <div class="jds-pagination__page-control--right">
+        <button
+          class="jds-pagination__navigation-button"
+          :disabled="disabled"
+          @click="onPreviousPage"
+        >
+          <jds-icon name="chevron-left" size="sm" />
+        </button>
+        <i class="jds-pagination__divider" />
+        <div class="jds-pagination__page-control__select">
+          <span>Halaman</span>
+          <select :value="mCurrentPage" @change="onPageChange" :disabled="disabled">
+            <option :value="page" v-for="page in generatedPageNumbers" :key="page">
+              {{ page }}
+            </option>
+          </select>
+          <span>dari <strong>{{ pages }}</strong></span>
+        </div>
+        <i class="jds-pagination__divider" />
+        <button class="jds-pagination__navigation-button" :disabled="disabled" @click="onNextPage">
+          <jds-icon name="chevron-right" size="sm" />
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import JdsIcon from '../JdsIcon'
+import localCopy from '../../mixins/local-copy'
+
+export default {
```

Comment:
can component value be set using `v-model`?. if so, please define `model` attribute :kiss:

Reply:
Okay, thank you!

Reply:
already fixed on this commit, @adrianpdm thank you! [80a603e](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/43/commits/80a603ed2662b700849f2eb8618f6faa9f542c0a)